src/pages/Settings/sections/ProfileSection.css

```
@@ -238,18 +246,31 @@
+  background: linear-gradient(135deg, #a0a0a0, #888888);
+  box-shadow: 0 3px 12px rgba(0,0,0,0.2);
+  transition: all 0.3s ease;
+}
+
+.profile-page .btn.primary.active {
+  background: linear-gradient(135deg, #55c5f1, #3aa8d4);
   box-shadow: 0 3px 12px rgba(85,197,241,0.3);
 }
 
-.profile-page .btn.primary:hover:not(:disabled) {
+.profile-page .btn.primary:hover:not(:disabled):not(.active) {
```

Stylelint (17.4.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)